File: module-server/src/main/java/de/twomartens/wahlrecht/model/dto/v1/VotingResult.java
package de.twomartens.wahlrecht.model.dto.v1;

import java.util.Map;

public record VotingResult(String electionName, String partyAbbreviation, String nominationName,
                           int votesOnNomination, int votesThroughHealing,
                           Map<Integer, Integer> votesPerPosition) {

}

File: module-server/src/main/kotlin/de/twomartens/wahlrecht/model/dto/v1/VotingResult.kt
package de.twomartens.wahlrecht.model.dto.v1

data class VotingResult(val electionName: String, val partyAbbreviation: String,
                        val nominationName: String,
                        val votesOnNomination: Int, val votesThroughHealing: Int,
                        val votesPerPosition: Map<Int, Int>)
